docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -962,16 +962,69 @@ You can configure the polling period by using the `spring.cloud.kubernetes.reloa
 It requires the same role as the monitored property source.
 This means, for example, that using polling on file-mounted secret sources does not require particular privileges.
 
+[[namespace-label-filtering]]
+=== Reload namespace and label filtering
+By default, a namespace chosen using the steps outlined in <<namespace-resolution,Namespace resolution>> will be used to listen to changes
+in configmaps and secrets. i.e.: if you do not tell reload what namespaces and configmaps/secrets to watch for,
+it will watch all configmaps/secrets from the namespace that will be computed using the above algorithm.
+
+On the other hand, you can define a more fine-grained approach. For example, you can specify the namespaces where
+changes will be monitored:
+
+====
+[source,yaml]
+----
+spring:
+  application:
+    name: event-reload
+  cloud:
+    kubernetes:
+      reload:
+        enabled: true
+        strategy: shutdown
+        mode: event
+        namespaces:
+          - my-namespace
+----
+====
+
+Such a configuration will make the app watch changes only in the `my-namespace` namespace. Mind that this will
+watch _all_ configmaps/secrets (depending on which one you enable). If you want an even more fine-grained approach,
+you can enable "label-filtering". First we need to enable such support via : `enable-reload-filtering: true`
+
+====
+[source,yaml]
+----
+spring:
+  application:
+    name: event-reload
+  cloud:
+    kubernetes:
+      reload:
+        enabled: true
+        strategy: shutdown
+        mode: event
+        namespaces:
+          - my-namespaces
+        monitoring-config-maps: true
+        enable-reload-filtering: true
+----
+====
+
+What this will do, is watch configmaps/secrets that only have the `spring.cloud.kubernetes.informer.enabled: true` label.
```

Comment:
I think it might be better to call the label `spring.cloud.kubernetes.config.informer.enabled`

Reply:
excellent point. done.